### ISSUE TITLE
Add the option to set availability to "Working Elsewhere"

### ISSUE THREAD

**Issue by probitas** [phw198/OutlookGoogleCalendarSync#1771]
Outlook now provides an additional state for calendar events "Working Elsewhere"

Within the SyncOptions "How" tab the dropdown for "Set availability to" has the following Options Free, Busy, Tentative, Out Of Office. Outlook now supports an additional option "Working Elsewhere".

please add this additional enumeration type

**Comment by probitas** [comment]
linked to enhancement #1505

**Comment by phw198** [comment]
> Outlook now provides an additional state for calendar events "Working Elsewhere"

Yes, in the client application, but not in the COM API. It's probable that the Graph API will support it though, and it can be looked at as part of [that adoption project](https://github.com/users/phw198/projects/2).
